### PR TITLE
*: add travis

### PR DESCRIPTION
**Pull request by runcom**
I guess this will be locked for now
ping @shanemhansen if you want to go with this :)

Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

**Comment by shanemhansen** [comment]
I'm stoked to get this flood of PR's. I won't really get a chance to review them until later today.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+go:
+  - 1.5.1
+  - 1.4.3
+  - 1.3.3
+
+before_install:
+  - go get golang.org/x/tools/cmd/vet
+# maybe one day
+#  - go get github.com/golang/lint/golint
```
Comment (vbatts) [inline .travis.yml]:
```bash
$ golint ./... | wc -l
3322
```

Comment (runcom) [inline .travis.yml]: damn

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+sudo: false
+
+script:
+  - go vet -x ./...
```
Comment (vbatts) [inline .travis.yml]: i guess we need to ensure openssl devel is present so that we can `go test ./...` ?

**Comment by vbatts** [comment]
@shanemhansen this is a flood! but we're excited to help!

File: .travis.yml (hunk 2 above)
Comment (runcom) [inline .travis.yml]: sure missed that, pkg-config would work right?

**Comment by shanemhansen** [comment]
I haven't messed with travis in a while. If this is ready for merging go ahead and merge it @vbatts. You should have permissions.

**Comment by runcom** [comment]
@shanemhansen you should activate travis for this repository and I guess only you can do this :(

**Comment by vbatts** [comment]
https://travis-ci.org/shanemhansen/gossl is for you @shanemhansen :-)

**Comment by shanemhansen** [comment]
I think I activated it.

**Comment by runcom** [comment]
@shanemhansen :tada: 

I guess I'll need to fix https://github.com/shanemhansen/gossl/pull/11#discussion_r40701883 tho

**Pull request merged**